docs/language/ztests/language-directed-acyclic-flow-graphs-1.yaml

```
@@ -0,0 +1,49 @@
+# ============================================================================
+# NOTE: This is a test to protect an example shown in the Zed language
+# docs. If you need to make changes to the Zed below to make the tests pass
+# again after a change, the docs likely need to be updated as well.
+#
+# See https://github.com/brimdata/zed/pull/2901 for details
+# ============================================================================
+
+# The Zed being tested below has been modified from the way it appears in
+# docs/language/README.md:
+#
+#   from PoolOne | split (
+#     => op1 | op2 | ... ;
+#     => op1 | op2 | ... ;
+#   ) | sort ts | ...
+#
+# Specifically, the "op1", "op2", and "..." have been filled in with real
+# operations.
+
+script: |
+  export ZED_LAKE_ROOT=test
+  zed lake init -q
+  zed lake create -q -p PoolOne
+  zed lake load -q -p PoolOne fruit.zson
+  zed lake query -z -I split.zed
+
+inputs:
+  - name: fruit.zson
+    data: |
+      {ts:2018-03-24T17:15:21Z,name:"apple",color:"red"}
+      {ts:2018-03-24T17:16:55Z,name:"banana",color:"yellow"}
+      {ts:2018-03-24T17:21:23Z,name:"avocado",color:"green"}
+      {ts:2018-03-24T17:23:29Z,name:"strawberry",color:"red"}
+  - name: split.zed
+    data: |
+      from PoolOne | split (
+        => every 5m allfruit:=collect(name),allcolor:=collect(color) | put numfruit:=len(allfruit) | put numcolor:=len(allcolor) ;
+        => put color:=to_upper(color) | put namelen:=len(name) | put tag:="mytag" ; 
```

I think we should adopt a Zed style in which a semicolon is not preceded by a space.
```suggestion
        => every 5m allfruit:=collect(name),allcolor:=collect(color) | put numfruit:=len(allfruit) | put numcolor:=len(allcolor);
        => put color:=to_upper(color) | put namelen:=len(name) | put tag:="mytag";
```

---

@nwt: Yes, "semicolon is not preceded by a space" is totally reasonable. I was being dense and blindly filling in the `...` and `op1` / `op2` stuff from the examples in the README and not removing the extra space chars that made sense _there_ to call attention to that stuff being placeholders and not real language elements. Will fix!